R/test-dm.R

```diff
@@ -1,15 +1,15 @@
-# Loads `dm` objects into one or more registered sources
-#
-# @description Works like `dbplyr::test_load()`, just for `dm`_objects.
-#
-# @param x
-#
-# @examples
-# dbplyr::test_register_src("df", dplyr::src_df(env = new.env()))
-# dbplyr::test_register_src("sqlite", dplyr::src_sqlite(":memory:", create = TRUE))
-#
-# cdm_test_obj <- cdm_nycflights13(cycle = TRUE)
-# cdm_test_obj_srcs <- cdm_test_load(cdm_test_obj)
+#' Loads `dm` objects into one or more registered sources
+#'
+#' @description Works like `dbplyr::test_load()`, just for `dm`_objects.
+#'
+#' @param x
+#'
+#' @examples
```

Comment:
This isn't meant to create documentation, but we can do the following:

```suggestion
#' @noRd
#' @examples
```